GliaWidgets/View/Chat/Message/Content/Choice/ChoiceCardOptionStyle.swift

```diff
@@ -10,6 +10,8 @@ public final class ChoiceCardOptionStyle {
 
     /// Style of an option in a disabled state - choice card has already been answered by the visitor or the choice card became inactive (e.g. engagement ended).
     public var disabled: ChatTextContentStyle
+    
+    public var disabledTabsBorderColor: UIColor
```

Comment:
I don't really like how it sticks out from the general structure. This style only has three properties, with each of them only describing their corresponding state. I feel like that should stay this way. 

So, I suggest subclassing the `ChatTextContentStyle` in, say `ChoiceCardOptionStateStyle` - which will only have an additional property `borderColor`. Maybe the integrator would like to change border colors of all choice card options' states, who knows - the more customization, the better. 

You can have this property be `nil` by default in the initializer or instead be explicitly set to `nil` from each state initializer (I also suggest breaking them up, more on that in later comments). This would make more sense semantically as well - `ChoiceCardOptionStyle` is a collection of three separate `ChoiceCardOptionStateStyle`s.

Reply:
...and don't forget to add description (documentation) to the new class, its property and init 😉 
You can contact me if you struggle with how it should be worded.